api/src/repositories/submission-job-queue-repository.ts

```diff
@@ -0,0 +1,96 @@
+import SQL from 'sql-template-strings';
+import { ApiExecuteSQLError } from '../errors/api-error';
+import { IProprietaryInformation } from '../services/submission-job-queue-service';
+import { BaseRepository } from './base-repository';
+
+export interface ISubmissionJobQueueModel {
+  submission_job_queue_id: number;
+  submission_id: number;
+  job_start_timestamp: string | null;
+  job_end_timestamp: string | null;
+  security_request: string | null; // stored as JSON object
```

Comment:
Could we rename this to `security_request_json`, if that's not too verbose?

Reply:
this model reflects the values in the database, so if we wanted to rename `security_request` it would require a migration.